<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>address</class>
 <widget class="QDialog" name="address">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Address</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="AddressCluster" name="_addr">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="activeVisible">
        <bool>true</bool>
       </property>
       <property name="listVisible" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>153</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="_buttonBox">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="_tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="_tabNotes">
      <attribute name="title">
       <string>Notes</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <widget class="XTextEdit" name="_notes">
         <property name="spellEnable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_tabComments">
      <attribute name="title">
       <string>Comments</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <widget class="Comments" name="_comments">
         <property name="type">
          <enum>Comments::Address</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_tabCharacteristics">
      <attribute name="title">
       <string>Characteristics</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <widget class="XTreeWidget" name="_charass"/>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="_newCharacteristic">
           <property name="text">
            <string>New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_editCharacteristic">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_deleteCharacteristic">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_tabUses">
      <attribute name="title">
       <string>Uses of the Address</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <widget class="XTreeWidget" name="_uses"/>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="_editAddrUse">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_viewAddrUse">
           <property name="text">
            <string>View</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="5"/>
 <customwidgets>
  <customwidget>
   <class>AddressCluster</class>
   <extends>QWidget</extends>
   <header>addresscluster.h</header>
  </customwidget>
  <customwidget>
   <class>Comments</class>
   <extends>QWidget</extends>
   <header>comments.h</header>
  </customwidget>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XTextEdit</class>
   <extends>QTextEdit</extends>
   <header>xtextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_tab</tabstop>
  <tabstop>_charass</tabstop>
  <tabstop>_newCharacteristic</tabstop>
  <tabstop>_editCharacteristic</tabstop>
  <tabstop>_deleteCharacteristic</tabstop>
  <tabstop>_uses</tabstop>
  <tabstop>_editAddrUse</tabstop>
  <tabstop>_notes</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_uses</sender>
   <signal>itemSelected(int)</signal>
   <receiver>_editAddrUse</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_uses</sender>
   <signal>valid(bool)</signal>
   <receiver>_editAddrUse</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_uses</sender>
   <signal>valid(bool)</signal>
   <receiver>_viewAddrUse</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_charass</sender>
   <signal>itemSelected(int)</signal>
   <receiver>_editCharacteristic</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_charass</sender>
   <signal>valid(bool)</signal>
   <receiver>_deleteCharacteristic</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_charass</sender>
   <signal>valid(bool)</signal>
   <receiver>_editCharacteristic</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
